src/_langs/en/tools/javascript/console/structured-data.markdown

```diff
@@ -3,8 +3,8 @@ rss: false
 layout: article
 title: "View Structured Data in Console"
 seotitle: "View Structured Data in Chrome DevTools Console"
-description: "TBD description."
-introduction: "TBD introduction."
+description: "Use the `table()` method to compare data objects."
```

Comment:
Invert sentence to push action first. Also, can we put full method name here and directly refer to the Chrome DevTools Console.